crates/ruff_linter/src/rules/pycodestyle/rules/logical_lines/whitespace_around_named_parameter_equals.rs

```diff
@@ -108,6 +109,8 @@ pub(crate) fn whitespace_around_named_parameter_equals(
         }
 
         match kind {
+            TokenKind::FStringStart => fstrings += 1,
```

Comment:
Nit: saturating add for parity with how we handle parens? Though shouldn't matter in practice.